wallet/concepts/multichain-api.md

```diff
@@ -0,0 +1,119 @@
+---
+description: Learn about the Multichain API.
+---
+
+# About the Multichain API
+
+:::tip Experimental
+The Multichain API is an experimental feature.
+:::
+
+The Multichain API is a scalable, generalized web3 wallet API that supports simultaneous
+interactions across multiple blockchain networks and ecosystems.
+When integrated with [MetaMask Snaps](/snaps), it enables developers to interact with both popular
+and emerging networks.
+Key benefits include:
+
+- **Elimination of chain switching** - The Multichain API allows dapps to interact with multiple EVM networks without having to request chain switches.
+  This feature significantly reduces development overhead involved with error handling.
+
+- **Extensibility** - The Multichain API can be integrated with
+  [interoperability Snaps](https://snaps.metamask.io/explore/), providing a standards-based interface
+  to non-EVM networks that can be broadly adopted across ecosystems.
+
+- **Seamless multichain UX** - The Multichain API offers improvements over EIP-1193 and [wallet-standard](https://github.com/wallet-standard/wallet-standard) interfaces.
+  It allows dapps to create unified multichain wallet connection flows, trigger transactions across disparate networks, and clearly interpret chain-specific addresses.
+
+[**Get started using the Multichain API.**](../how-to/manage-networks/use-multichain.md)
+
+## Technical overview
+
+The Multichain API follows the [CAIP-25](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-25.md)
+standard for dapps to interface with multichain wallets.
+See [MIP-5](https://github.com/MetaMask/metamask-improvement-proposals/blob/main/MIPs/mip-5.md) and
+[MIP-6](https://github.com/MetaMask/metamask-improvement-proposals/blob/main/MIPs/mip-6.md) for
+detailed information about MetaMask's Multichain API implementation.
+
+The API includes a method [`wallet_createSession`](../reference/multichain-api.md#wallet_createsession)
+that dapps can call to create a multichain connection with a wallet, with specified properties and
+authorization scopes.
+Dapps can update the connection using the same method `wallet_createSession`.
+
+Dapps can use [`wallet_invokeMethod`](../reference/multichain-api.md#wallet_invokemethod) to
+call a subset of the [Wallet JSON-RPC API methods](../reference/json-rpc-methods/index.md) on
+a specified chain.
+Dapps can use [`wallet_getSession`](../reference/multichain-api.md#wallet_getsession) to get
+the scopes and properties of the active connection, and use
+[`wallet_revokeSession`](../reference/multichain-api.md#wallet_revokesession) to revoke the connection.
+The API also supports the following events:
+
+- [`wallet_notify`](../reference/multichain-api.md#wallet_notify) notifies dapps of onchain events or state changes they previously subscribed to.
+- [`wallet_sessionChanged`](../reference/multichain-api.md#wallet_sessionchanged) notifies dapps of changes to the multichain connection.
```

Comment:
Perhaps we can improve readability here?
Maybe something like, "The API incudes methods for managing multichain connections between dapps and wallets:", then list `wallet_createSession`, `wallet_invokeMethod` in a list format?